kuma/authkeys/migrations/0003_remove_authkeys.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        orm.Key.objects.all().delete()
```

Comment:
Ah, I thought for some reason that South's fake ORM wouldn't trigger the default cascade delete, sorry.

Yeah, I think changing it to only delete the really unused keys makes sense +1